### PR TITLE
Use radius instead of max_radius in delete chunks.

### PR DESCRIPTION
**Pull request by nsg**
This PR uses the current radius instead of the static max_radius.

@petterarvidsson ready for review

### CODE REVIEW

**Comment by petterarvidsson** [comment]
I think it would make sense to have the same rule in all cases. Now chunk data is r + 2 and the rest is just r. Therefore the chunk data saved will not be used (other than weirdly showing up in collision detection and selection). Maybe + 1 is a good rule for all three cases?

**Comment by nsg** [comment]
@petterarvidsson yes, agree!

I found a interesting performance difference with this code. I have a hard time to get over 4 in view distance with this, not sure why. Need to investigate this before merge. I never noticed this because it was not noticeable in #192

**Comment by nsg** [comment]
I have rebased with +1

File: src/chunk_shader.cpp [hunk 1]
```diff
@@ -99,7 +98,7 @@ namespace konstructs {
                 matrix::ext_frustum_planes(planes, radius, m);
                 for(auto it = models.begin(); it != models.end();) {
                     int distance = (it->second->position - player_chunk).norm();
-                    if ( distance > max_radius) {
+                    if (distance > radius) {
```
Comment (petterarvidsson) [inline src/chunk_shader.cpp]: @nsg I would use + 1 here as well

File: src/client.cpp [hunk 2]
```diff
@@ -400,7 +400,7 @@ namespace konstructs {
                     request_chunks = 1;
                 }
                 for(auto it = received.begin(); it != received.end();) {
-                    if((*it - p_chunk).norm() > max_radius) {
+                    if((*it - p_chunk).norm() > r) {
```
Comment (petterarvidsson) [inline src/client.cpp]: @nsg and here

File: src/client.cpp [hunk 3]
```diff
@@ -400,7 +400,7 @@ namespace konstructs {
                     request_chunks = 1;
                 }
                 for(auto it = received.begin(); it != received.end();) {
-                    if((*it - p_chunk).norm() > r) {
+                    if((*it - p_chunk).norm() > r + 1) {
```
Comment (nsg) [inline src/client.cpp]: If we are at view distance 8 and drops down to 7, this change will make sure that we still have layer 8 loaded. If not we will drop payer 8 and ask for it a few ms later.

File: src/client.cpp [hunk 4]
```diff
@@ -421,7 +421,7 @@ namespace konstructs {
                                 if(received.find(pos) == received.end()) {
                                     if(requested.find(pos) == requested.end()) {
                                         /* We don't have it and didn't request it */
-                                        if(distance < best_chunk_distance) {
+                                        if(distance < best_chunk_distance && distance < r) {
```
Comment (nsg) [inline src/client.cpp]: The corners are outside the view distance, so we will ask for chunks and drop them all the time.

**Comment by nsg** [comment]
@petterarvidsson updated

**Comment by petterarvidsson** [comment]
Fixed by #192

**Pull request closed without merging**